### PR TITLE
Added sponsorship-prospectus.html page

### PR DESCRIPTION
**Pull request by raukadah**
Signed-off-by: Chandan Kumar (raukadah) <raukadah@gmail.com>

### CODE REVIEW

File: _data/sponsorship.yml [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+    gold: Half Page
+    silver: no
+    startup: no
+  - type: Direct marketing Video ads (up to 20 seconds)
+    diamond: 8
+    platinum: 6
+    gold: 2
+    silver: 1 (10 sec)
+    startup: no
+  - type: Virtual Swag: PDF
```
Comment (palnabarun) [inline _data/sponsorship.yml]: In YAML, you have to encapsulate values containing `:` with quotes.

```suggestion
  - type: "Virtual Swag: PDF"
```

File: _data/sponsorship.yml [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+    diamond: yes
+    platinum: yes
+    gold: yes
+    silver: yes
+    startup: yes
+  - type: Virtual Swag: URL + coupon codes
```
Comment (palnabarun) [inline _data/sponsorship.yml]: Same as above.
```suggestion
  - type: "Virtual Swag: URL + coupon codes"
```

File: _includes/supporter.html [hunk 3]
```diff
@@ -10,8 +10,8 @@
       and maintain the inventory for the conference.
     </div>
     <p class="description">
-      Reach out to us at
-      <a href="mailto:sponsor@in.pycon.org">sponsor@in.pycon.org</a>
+      Take a look at
+      <a href="{{ '/sponsorship-prospectus' | relative_url }}">Sponsorship Page</a>
```
Comment (palnabarun) [inline _includes/supporter.html:14]: Can we rename the page to `/sponsor`?

File: sponsorship-prospectus.md [hunk 4]
```diff
@@ -0,0 +1,186 @@
...
+Participant details will not be shared without their explicit permission.
+Participants are allowed to share their information with you at your stall.
+Give them a good reason to do so. Make a compelling pitch for yourself 
+or offer goodies in exchange for contact information.
+
+<br>
+
+## Contact
+
+- Mail: [sponsorship@in.pycon.org](mailto:sponsorship@in.pycon.org)
```
Comment (palnabarun) [inline sponsorship-prospectus.md]: The email ID for sponsorship team is actually `sponsor@in.pycon.org`.
```suggestion
- Mail: [sponsor@in.pycon.org](mailto:sponsor@in.pycon.org)
```

File: sponsorship-prospectus.md [hunk 5]
```diff
@@ -0,0 +1,186 @@
...
+- *Abhishek Mishra*, Sponsorship Coordinator
+  - Mobile: +91-70664 87364
+- *Chandan Kumar*, Sponsorship Coordinator
+  - Mobile: +91- 96072 64082
+- *Sayan Chowdhury*, PyCon India 2020 Chair
+  - Mobile: +91-96869 92532
```
Comment (palnabarun) [inline sponsorship-prospectus.md]: Are you sure with putting out the phone numbers in public?

**Review by palnabarun — CHANGES_REQUESTED** [review]
Left a few comments.

Also, we didn't link to the sponsorship-prospectus PDF anywhere in the page? Shouldn't we put a link to that too?

**Comment by palnabarun** [comment]
The Netlify build is also failing due to the unquoted values containing `:` in `data/sponsorship.yml`.

File: _includes/supporter.html (hunk 3 above)
Comment (raukadah) [reply _includes/supporter.html:14]: /sponsor page will be used for listing out the sponsors.

File: sponsorship-prospectus.md (hunk 5 above)
Comment (raukadah) [reply sponsorship-prospectus.md]: Yes, good point, removing that.

File: _data/sponsorship.yml (hunk 2 above)
Comment (raukadah) [reply _data/sponsorship.yml]: Great, learnt something new about quotes

**Review by raukadah — COMMENTED** [review]
Thank you for the reviews. Suggested changes are added.

File: _includes/supporter.html [hunk 6]
```diff
@@ -10,9 +10,12 @@
...
       to support PyCon India 2020.
     </p>
+    <div class="text-center" style="position: relative;">
+      <a href="/2020/sponsorship-prospectus.pdf" class="dwd-link"><b>DOWNLOAD SPONSORSHIP PROSPECTUS</b></a>
```
Comment (Sc0rpi0n101) [inline _includes/supporter.html]: How about having this as a button?

```suggestion
      <a class="primary-button" rel="nofollow noreferrer" href="/2020/sponsorship-prospectus.pdf" target="_blank">
        Sponsorship Prospectus
      </a>
```

File: sponsorship-prospectus.md [hunk 7]
```diff
@@ -0,0 +1,180 @@
...
+    </tr>
+    <tr>
+      <td>Startup</td>
+      <td>₹ 50,000</td>
+      <td>$ 700</td>
+    </tr>
+  </tbody>
+</table>
+
+The above mentioned sponsorship amount is exclusive of 18% GST.
```
Comment (Sc0rpi0n101) [inline sponsorship-prospectus.md]: This should be in bold imo.
```suggestion
**The above mentioned sponsorship amount is exclusive of 18% GST.**
```

File: sponsorship-prospectus.md [hunk 8]
```diff
@@ -0,0 +1,180 @@
...
+domains and expertise which includes and not limited to -
+Developers, technologists, programmers, educators, bloggers,
+authors, web developers, CTOs, managers, entrepreneurs,
+scientists, engineers and domain experts.
+
+<br>
+
+## Sponsorship Packages
+
+<table class="table table-striped">
```
Comment (Sc0rpi0n101) [inline sponsorship-prospectus.md]: The table columns are rendering too close in the netlify build. It looks kinda odd. 
Also, it looks like plaintext. Not table rows. There are no borders in the table.

File: sponsorship-prospectus.md [hunk 9]
```diff
@@ -0,0 +1,180 @@
...
+  </tbody>
+</table>
+
+The above mentioned sponsorship amount is exclusive of 18% GST.
+
+<br>
+
+## Sponsorship Benefits
+
+<table class="table table-striped">
```
Comment (Sc0rpi0n101) [inline sponsorship-prospectus.md]: Same here, this table is also rendering without any borders in the current build. Also, the Startup stuff is missing.

Comment (raukadah) [reply sponsorship-prospectus.md]: startup part is fixed now.

File: sponsorship-prospectus.md (hunk 8 above)
Comment (raukadah) [reply sponsorship-prospectus.md]: I have no idea how to add border in the table using jekyll. Do we want add table, th, td {
  border: 1px solid black;
  border-collapse: collapse;
} in assets/css/style.css or something else?

**Review by raukadah — COMMENTED** [review]
Made the required changes, need help on border part

**Review by raukadah — COMMENTED** [review]
Add changes done

**Review by palnabarun — APPROVED** [review]

**Review by sayanchowdhury — APPROVED** [review]

**Pull request merged**